testplan/web_ui/testing/src/Report/BatchReport/components/PrintButton.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
```

Comment:
PrintButton is just a simple component which contains a link to download a PDF file. Does it need connector?

Reply:
Nope. [Current version](https://github.com/Morgan-Stanley/testplan/pull/451/files#diff-72a1cd9a687c8417d80b6ac9a75147fd) doesn't use it.